web/packages/api/src/parachains/neuroweb.ts

```diff
@@ -0,0 +1,134 @@
+import { ParachainBase } from "./parachainBase"
+import { DOT_LOCATION, erc20Location } from "../xcmBuilder"
+import { PNAMap } from "../assets_v2"
+import { AssetMap } from "@snowbridge/base-types"
+import { ApiPromise } from "@polkadot/api"
+import { SubmittableExtrinsic } from "@polkadot/api/types"
+import { ISubmittableResult } from "@polkadot/types/types"
+
+export const NEUROWEB_TEST_CHAIN_ID = 11155111 // Sepolia
```

Comment:
Maybe inject `ethChainId` (already in the registry) into the `ParachainBase` constructor, or move this const into a common class?

Reply:
Its just relevant to Neuroweb, and Mythos config is defined the same way: https://github.com/Snowfork/snowbridge/blob/main/web/packages/api/src/parachains/mythos.ts#L6-L9 Since it is unlikely to change and this is a small class already, it think its better to leave it here.